### ISSUE TITLE
Accuracy metrics

### ISSUE THREAD

**Issue by umarikkar** [fuying-wang/MGCA#12]
Hi,

Can I know at which point the pre-training usually stops? I used the default settings (same GPUs and batch size) and the final saved cpt was *epoch=29-step=47339.ckpt*. Should I use this? or *last.ckpt* for the checkpoint?

When using the former, I was able to achieve a fine-tuning AUC of 86.98% on 0.01% of CheXpert. Could you please let me know if I am missing something here? I have attached the args file and training output screenshots for reference.

[sysout_error.txt](https://github.com/fuying-wang/MGCA/files/10676418/sysout_error.txt)
[sysout_terminal.txt](https://github.com/fuying-wang/MGCA/files/10676419/sysout_terminal.txt)
[args.txt](https://github.com/fuying-wang/MGCA/files/10676425/args.txt)

Thanks!

**Comment by fuying-wang** [comment]
Hi, 

You may check the loss curve and find the epoch with the smallest valid loss.
